frontend/src/hooks/datetime-hooks.ts

```diff
@@ -14,3 +15,46 @@ export function useDateTimeFormatter(): Function {
     [settings]
   );
 }
+
+export function CalculateDuration(date: Date, t: TFunction): string {
```

Comment:
dayjs supports this out of the box:

https://day.js.org/docs/en/display/from-now

Reply:
Switched to dayjs

Reply:
Switched to dayjs